geode-core/src/main/java/org/apache/geode/management/internal/i18n/CliStrings.java

```diff
@@ -1653,6 +1653,23 @@
   public static final String LIST_FUNCTION__NO_FUNCTIONS_FOUND_ERROR_MESSAGE = "No Functions Found";
 
   public static final String LIST_GATEWAY = "list gateways";
+  public static final String LIST_GATEWAY__SHOW_SENDERS_ONLY = "senders-only";
+  public static final String LIST_GATEWAY__SHOW_SENDERS_ONLY_INPUT_ERROR =
+      "senders-only must be 'true' or 'false'";
+  public static final String LIST_GATEWAY__SHOW_SENDERS_ONLY__HELP =
+      "Display gateway senders only.";
+  public static final String LIST_GATEWAY__SHOW_RECEIVERS_ONLY = "receivers-only";
+  public static final String LIST_GATEWAY__SHOW_RECEIVERS_ONLY_INPUT_ERROR =
+      "receivers-only must be 'true' or 'false'";
+  public static final String LIST_GATEWAY__SHOW_RECEIVERS_ONLY__HELP =
+      "Display gateway receivers only.";
+
+  public static final String LIST_GATEWAY__SHOW_PARAM_TRUE = "true";
+  public static final String LIST_GATEWAY__SHOW_PARAM_FALSE = "false";
+  public static final String LIST_GATEWAY__SHOW_PARAM_EMPTY = "empty";
```

Comment:
I don't think these new constants are needed and, actually, I believe it makes things harder to understand than before... (see my other comments).